From de405d3810dcd1c187c03b9848f51814098cc04d Mon Sep 17 00:00:00 2001
From: hrzndelta <hrzndelta@proton.me>
Date: Wed, 1 Oct 2025 13:50:04 +0800
Subject: [PATCH 2/2] Bootanimation: add a switch to replace it with google
 monet

---
 config/common.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/config/common.mk b/config/common.mk
index 06749ff0..684fb995 100644
--- a/config/common.mk
+++ b/config/common.mk
@@ -122,6 +122,7 @@ endif
 # Bootanimation
 TARGET_SCREEN_WIDTH ?= 1080
 TARGET_SCREEN_HEIGHT ?= 1920
+ifneq ($(TARGET_SHIP_MONET_BOOTANIMATION),true)
 PRODUCT_PACKAGES += \
     bootanimation.zip \
     bootanimation-dark.zip
-- 
2.51.0

